defi/src/storeTvlInterval/computeTVL.ts

```diff
@@ -215,6 +227,100 @@ async function getTokenData(readKeys: string[], timestamp: string | number): Pro
   }
 }
 
+const mcapCache: { [PK: string]: any } = {}
+
+async function getMcapData(readKeys: string[], timestamp: string | number): Promise<any> {
```

Comment:
looks good, can you move this and the price function to `coinsApi.ts` in the same folder, these functions can be used elsewhere as well which have nothing to do with tvl

Reply:
hmm, on second thought, if you use latest sdk version, it already has functions that return cached price & mcap info?